src/server/config/swagger.json

```diff
@@ -485,74 +485,129 @@
-          "get": {
-            "tags": ["default values"],
-            "summary": "Get default values for specific batch by batchId",
-            "description": "",
-            "operationId": "getDefaultValues",
-            "produces": ["application/json"],
-            "parameters": [
-              {
-                "name": "authorization",
-                "in": "header",
-                "description": "Firebase token",
-                "required": true,
-                "type": "string",
-                "default": "Bearer ENTER_FIREBASE_TOKEN"
-              },
-              {
-                "name": "batchId",
-                "in": "path",
-                "type": "integer",
-                "required": true
-              },
-              {
-                "name": "stage",
-                "in": "query",
-                "type": "string",
-                "required": false
-              }
-            ],
-            "responses": {
-              "200": {
-                "description": "Default values returned."
-              },
-              "401": {
-                "description": "You are not authorized to view this content."
-              },
-              "404": {
-                "description": "A batch with the specified id was not found."
-              },
-              "default": {
-                "description": "Unexpected error."
-              }
-            },
-            "security": [
-              {
-                "fireabse_auth": ["read"]
-              }
-            ]
+        "security": [
+          {
+            "fireabse_auth": ["read"]
           }
-        }
-      },
-      "securityDefinitions": {
-        "firebase_auth": {
-          "type": "oauth2",
-          "authorizationUrl": "seasony-login.firebaseapp.com",
-          "flow": "accessCode",
-          "scopes": {
-            "read": "read data based on user uid",
-            "write": "write data based on user uid"
+        ]
+      }
+    },
+    "/batches/all": {
+      "get": {
+        "tags": ["all batches"],
+        "parameters": [
+          {
+            "name": "authorization",
+            "in": "header",
+            "description": "Firebase token",
+            "required": true,
+            "type": "string",
+            "default": "Bearer ENTER_FIREBASE_TOKEN"
+          }
+        ],
+        "summary": "get list of all batches together with current stage and status",
+        "description": "get list of all batches together with current stage and status",
+        "operationId": "getAllBatches",
+        "produces": ["application/json"],
+        "responses": {
+          "200": {
+            "description": "Successful request"
+          },
+          "401": {
+            "description": "Unauthorized request"
+          },
+          "404": {
+            "description": "Not found"
+          },
+          "500": {
+            "description": "Server error"
           }
         }
       }
+    },
```

Comment:
You can put this in the route file so it follows the same way @ricardoaguiar wrote the docs [here](https://github.com/HackYourFuture-CPH/FP-class11/pull/184) because I think we won't keep this file...